File: src/main/resources/views/aether/src/entity/dto/GetGroupListDto.ts
import type PageQuery from "@/entity/PageQuery.ts";


export default interface GetGroupListDto extends PageQuery{

    // 模糊匹配 组标识、组名称、组描述
    keyword?: string;

    // 组状态：0:禁用 1:启用
    status?: number;
    
} 

File: src/main/java/com/ksptool/ql/biz/model/dto/GetGroupListDto.java
package com.ksptool.ql.biz.model.dto;

import com.ksptool.ql.commons.web.PageQuery;
import lombok.Data;
import lombok.EqualsAndHashCode;

@EqualsAndHashCode(callSuper = true)
@Data
public class GetGroupListDto extends PageQuery {

    //模糊匹配 组标识、组名称、组描述
    private String keyword;

    //组状态：0:禁用 1:启用
    private Integer status;

}
